lib/Driver/ToolChains.cpp

```diff
@@ -1147,11 +1163,25 @@ toolchains::GenericUnix::constructInvocation(const LinkJobAction &job,
   }
   llvm::sys::path::append(RuntimeLibPath,
                           getPlatformNameForTriple(getTriple()));
+
+  Arguments.push_back(context.Args.MakeArgString(
```

Comment:
Please add a comment here saying what this is doing and why it needs to come first. (I just spent a bit convincing myself of that.)